### PR TITLE
Remove support for exporting ViewData

### PR DESCRIPTION
**Pull request by songy23**
Library export now uses Metric. ViewData isn't being used now.

### CODE REVIEW

**Comment by bogdandrutu** [comment]
Please rebase

File: stackdriver.go [hunk 1]
```diff
@@ -343,9 +343,8 @@ func NewExporter(o Options) (*Exporter, error) {
 
 // ExportView exports to the Stackdriver Monitoring if view data
 // has one or more rows.
-func (e *Exporter) ExportView(vd *view.Data) {
-	e.statsExporter.ExportView(vd)
-}
+// Deprecated: this method does not have any effect now, use ExportMetrics instead.
+func (e *Exporter) ExportView(vd *view.Data) {}
```
Comment (bogdandrutu) [inline stackdriver.go]: Remove this, prefer to break build rather than be a no-op :)

Comment (songy23) [reply stackdriver.go]: Make sense. I expect the effect to be minimal since we switched to exporting metrics long ago.

**Comment by rghetia** [comment]
we should  update  following examples.
1. https://opencensus.io/integrations/google_cloud/google_cloud_spanner/go/
2 .https://github.com/googleapis/google-cloud-go/wiki/OpenCensus
3. https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/4b7f30648a4f2cf529cecb4274f016b9cc30df7e/examples/stats/main.go#L49 - should call exporter.start*/stop*
4. https://opencensus.io/exporters/supported-exporters/go/stackdriver/

**Pull request merged**